src/audio-source.ts

```diff
@@ -1,11 +1,14 @@
-import {Component, WonderlandEngine} from '@wonderlandengine/api';
+import {Component, Emitter, WonderlandEngine} from '@wonderlandengine/api';
 import {property} from '@wonderlandengine/api/decorators.js';
-import {
-    _audioContext,
-    AudioListener,
-    getAudioData,
-    audioBuffers,
-} from './audio-listener.js';
+import {_audioContext, AudioListener, _unlockAudioContext} from './audio-listener.js';
+import {AudioChannel, AudioManager, PlayState} from './audio-manager.js';
+import {MIN_RAMP_TIME, MIN_VOLUME} from './audio-players.js';
+
+export enum PanningType {
+    NONE,
+    REGULAR,
+    HRTF,
```

Comment:
This looks promising and is related to my comment on the Readme :)

Just fix the CamelCasing here 😌